### PR TITLE
os/bluestore: fix duplicate allocations in bmap allocator

### PR DESCRIPTION
**Pull request by ifed01**
Fixes: http://tracker.ceph.com/issues/40080

Signed-off-by: Igor Fedotov <ifedotov@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/os/bluestore/BitmapAllocator.cc [hunk 1]
```diff
@@ -36,7 +36,7 @@ int64_t BitmapAllocator::allocate(
   }
   for (auto e : *extents) {
     ldout(cct, 10) << __func__
-                   << " 0x" << std::hex << e.offset << "~" << e.length
+                   << "extent: 0x" << std::hex << e.offset << "~" << e.length
```
Comment (liewegas) [inline src/os/bluestore/BitmapAllocator.cc]: missing space between e.length and 'extent'?

**Comment by tchaikov** [comment]
retest this please.

**Comment by ifed01** [comment]
retest this please.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2019-06-16_17:08:46-rados-wip-kefu-testing-2019-06-16-2148-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2019-06-18_09:55:35-rados-wip-kefu-testing-2019-06-16-2148-distro-basic-smithi/

failures already tracked/addressed by

- http://tracker.ceph.com/issues/17882
- 2dd5574cf9a5a80cc40a365675bb1e33a32c9f41

**Pull request merged**